src/transactions/TransactionFrame.cpp

```diff
@@ -273,39 +296,42 @@ TransactionFrame::commonValidPreSeqNum(Application& app, LedgerDelta* delta)
 }
 
 void
-TransactionFrame::processSeqNum(LedgerManager& lm, LedgerDelta& delta)
+TransactionFrame::processSeqNum(AbstractLedgerState& ls)
 {
-    if (lm.getCurrentLedgerVersion() >= 10)
+    auto header = ls.loadHeader();
+    if (header.current().ledgerVersion >= 10)
     {
-        if (mSigningAccount->getSeqNum() > mEnvelope.tx.seqNum)
+        auto sourceAccount = loadSourceAccount(ls, header);
+        if (sourceAccount > mEnvelope.tx.seqNum)
```

Comment:
bug: should be `sourceAccount.current().data.account().seqNum`
on that note: why do we need the `bool()` operator?
I tend to think that implicit conversions are evil - which includes non explicit constructors that take a single parameter (mmm, doing a sweep)